ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -513,40 +513,40 @@ object Config {
           )
 
         // TODO append-only: remove after removing support for the current (mutating) schema
-        opt[Unit]("index-append-only-schema-unsafe")
+        opt[Unit]("index-append-only-schema")
           .optional()
           .hidden()
           .text(
-            s"Use the append-only index database with parallel ingestion. Highly unstable. Should not be used in production."
+            s"Use the append-only index database with parallel ingestion."
           )
           .action((_, config) => config.copy(enableAppendOnlySchema = true))
 
-        opt[Unit]("mutable-contract-state-cache-unsafe")
+        opt[Unit]("mutable-contract-state-cache")
           .optional()
           .hidden()
           .text(
-            "Experimental contract state cache for command execution. Must be enabled in conjunction with index-append-only-schema-unsafe. Should not be used in production."
+            "Contract state cache for command execution. Must be enabled in conjunction with index-append-only-schema."
```

Comment:
If these must be used in conjunction, why not have a single command-line option e.g. `indexer-enable-parallel-ingestion`?

Reply:
They don't have to be used in conjunction, append-only is fine to be set alone. It is a prerequisite for mutable-state-cache